### ISSUE TITLE
How to generate tflite file for object detection on android in tensorflow 2.3

### ISSUE THREAD

**Issue by vmishra2019** [tensorflow/tensorflow#48011]
I am using SSD pretrained model ssd_mobilenet_v1_fpn_640x640_coco17_tpu-8, I trained it on tensorflow 2.3, after checkpoint has been generated i have downloaded it on my windows machine.

Can someone please share scripts require to generate tflite file for android, also following scripts works for me in tensorflow 1.13 which is not supported in 2.x

python object_detection/export_tflite_ssd_graph.py --pipeline_config_path=pipeline.config --trained_checkpoint_prefix=model.ckpt-4000 --output_directory=saved_model --add_postprocessing_op=true
	
tflite_convert --graph_def_file=tflite_graph.pb --output_file=saved_model.tflite --input_shapes=1,300,300,3 --input_arrays=normalized_input_image_tensor --output_arrays=TFLite_Detection_PostProcess,TFLite_Detection_PostProcess:1,TFLite_Detection_PostProcess:2,TFLite_Detection_PostProcess:3 --default_ranges_min=0 --default_ranges_max=255 --inference_type=QUANTIZED_UINT8 --mean_values=128 --std_dev_values=128 --change_concat_input_ranges=false --allow_custom_ops

**Comment by abattery** [comment]
Could you leave your question at the tensorflow/model repository, which owns the objection_detector scripts?

**Comment by vmishra2019** [comment]
How can i move it to tensorflow/models

**Comment by abattery** [comment]
Maybe you can just upload a new issue at there and close.
